uiowa/uiowa
# [Facilities] Structured titles for alerts

# How to test

- Code review 

## Test on facilities
```
ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local uli node/add/alert
```

1. Test out using only the project title with building set to none
2. Test using Project title and building field
3. Test with building field set to -none- and check the "Override the building name in the page title" field and provide override text in the "Location description" field
4. Test with building field set to a building and check the "Override the building name in the page title" field and provide override text in the "Location description" field
5. Verify category is now a checkbox 
6. Verify building field is a select list
7. Verify body field label has changed to "Reason and details for closure" and set to Minimal plus format
8. Unpublish should be removed from Scheduling options

## Test on default
```
ddev blt ds --site=default && ddev drush @default.local config-split:activate sitenow_alerts -y && ddev drush @default.local uli node/add/alert
```
- Verify that unpublished option is gone and that no other fields are changed from the above facilities work.
